nexus/pkg/artifacts/downloader.go

```diff
@@ -27,7 +27,6 @@ type ArtifactDownloader struct {
 	// Input
 	QualifiedName          string
 	DownloadRoot           *string
-	Recursive              *bool
 	AllowMissingReferences *bool
 	// Properties
 	UpstreamArtifacts   []gql.ArtifactByIDArtifact
```

Comment:
You can probably delete `UstreamArtifacts` too then. I had added it here to populate it with the  artifacts that this artifact depends on (similar to `artifact._dependent_artifacts` in the python code). It was ultimately going to be used to download artifacts recursively.